Mechanic2.roboFontExt/info.plist

```diff
@@ -34,6 +34,6 @@
 	<key>timeStamp</key>
 	<real>1527887300.4018419</real>
 	<key>version</key>
-	<string>1.2</string>
+	<string>1.32</string>
```

Comment:
a jump to 1.3 is fine, unless you step by 31 :)

Reply:
Hehe, that’s because I had a v1.3, then a 1.31, the 1.32 is what was left. But yeah, let me bump to 1.3!